joblib/_utils.py

```diff
@@ -0,0 +1,37 @@
+# Adapted from https://stackoverflow.com/a/9558001/2536294
+
+import ast
+import operator as op
+
+# supported operators
+operators = {
+    ast.Add: op.add,
+    ast.Sub: op.sub,
+    ast.Mult: op.mul,
+    ast.Div: op.truediv,
+    ast.Pow: op.pow,
+    ast.USub: op.neg,
+}
+
+
+def eval_expr(expr):
+    """
+    >>> eval_expr('2*6')
+    12
+    >>> eval_expr('2**6')
+    64
+    >>> eval_expr('1 + 2*3**(4) / (6 + -7)')
+    -161.0
+    """
+    return eval_(ast.parse(expr, mode="eval").body)
```

Comment:
Let's make it more user friendly:

```suggestion
    try:
        return eval_(ast.parse(expr, mode="eval").body)
    except TypeError:
        raise ValueError(f"{expr!r} is not a valid arithmetic expression.")
```

Reply:
The tests will have to be adapted accordingly.